collector/README.md

```diff
@@ -61,93 +61,96 @@ marked with a '?' in the `compare` page.
-measurements as Cargo sometimes begins passing (or stops passing) various flags
-to rustc.
+- `<RUSTC>`: a path (relative or absolute) to a rustc executable that will be
+  benchmarked. Some benchmarks use procedural macros, which require a stage 2
+  compiler. Therefore, the value is likely to be something like
+  `$RUST/build/x86_64-unknown-linux-gnu/stage2/bin/rustc`, where `$RUST` is a
+  path (relative or absolute) to a `rust` repository.
 
-`$ID` is an identifier which will be used to identify the results in the
-collected data.
-
-The full benchmark suite takes some time to run: tens of minutes or more,
-depending on the speed of your machine. Progress output is printed to stderr.
-
-`RUST_LOG=debug` can be specified to enable some verbose logging, which is
-useful for debugging rustc-perf itself.
+- `<ID>`: an identifier which will be used to identify the results in the
+  database.
 
 ### Benchmarking options
 
-The following options, if present, must appear before `bench_local` in the
-command.
-- `--self-profile` can be used to do self-profiling, which requires the
-  `measureme` tool.
-- `--filter $STR` can be used to run a subset of the benchmarks. `$STR` is a
-  substring of the name of the benchmark(s) you wish to run.
-- `--exclude $STR` is the inverse of `--filter`. `$STR` is a substring of the
-  name of the benchmark(s) you wish to skip.
-
-The following options, if present, must appear after `bench_local` in the
-command.
-- `--builds $BUILDS` can be used to select what kind of builds are profiled.
-  The possible choices are one or more (comma-separated) of `Check`, `Debug`,
-  `Opt`, and `All` (the default).
-- `--runs $RUNS` can be used to select what profiling runs are done for each
-  build. The possible choices are one or more (comma-separated) of `Full`,
-  `IncrFull`, `IncrUnchanged`, `IncrPatched`, and `All` (the default). Note
-  that `IncrFull` is always run (even if not requested) if either of
-  `IncrUnchanged` or `IncrPatched` are run.
-
-### Comparing different versions on your own machine
+The following options alter the behaviour of the `bench_local` subcommand.
+- `--builds <BUILDS>`: the build kinds to be benchmarked. The possible choices
+  are one or more (comma-separated) of `Check`, `Debug`, `Doc`, `Opt`, and
+  `All`. The default is `Check,Debug,Opt`.
+- `--cargo <CARGO>`: a path (relative or absolute) to a Cargo executable that
+  will be used to build the benchmarks. By default, the nightly Cargo installed
+  by `rustup` will be used. This is usually fine, though in rare cases it may
+  cause local results to not exactly match production results, because Cargo
+  sometimes begins passing (or stops passing) various flags to rustc.
+- `--db $DATABASE`: a path (relative or absolute) to a sqlite database file in
+  which the timing data will be placed. It will be created if it does not
+  already exist. The default is `results.db`. Alternatively, the collector
```

Comment:
Maybe worth mentioning that results.db is a sqlite file.

Reply:
Maybe name it `result.sqlite` so it's obvious?

Reply:
I will mention the sqlite-ness, but leave the `.db` suffix -- it's shorter, matches the `--db` option, and future-proofs against possible future format changes :)